src/rise-twitter.js

```diff
@@ -113,7 +113,7 @@ export default class RiseTwitter extends HTMLElement {
         this._play();
       } else {
         console.log('_handlePlay NOT IsAuthorized');
-        this.eventHandler.emitDone();
+        this.tweet.handleError();
```

Comment:
@rodrigopavezi As discussed this shouldn't be calling `tweet.handleError()`, thanks!

Reply:
@jungeunyoon yeah, I think the bug is in another place. Checking it now. cheers